### PR TITLE
feat: update based on trivet-monolingual@1.x 

### LINKED ISSUE #313: Use eleventy-plugin-fluid for asset processing

### LINKED ISSUE #314: Add site map

### LINKED ISSUE #315: Add news RSS feed

### LINKED ISSUE #316: Update linting based on Trivet

### LINKED ISSUE #317: Update Markdown config using eleventy-plugin-fluid conventions

### PR DESCRIPTION
**Pull request by greatislander**
- [x] feat: move output to _site
- [x] ci: update workflows
- [x] chore(deps): add Dependabot configuration
- [x] feat: use eleventy-plugin-fluid for asset processing (resolves #313)
- [x] feat: update Markdown config (resolves #317)
- [x] feat: add sitemap (resolves #314)
- [x] feat: use Trivet-style linting (resolves #316)

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+updates:
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: monthly
+    allow:
+      - dependency-type: direct
+    ignore:
+      - dependency-name: stylelint
+        versions: [^15]
```
Comment (chosww) [inline .github/dependabot.yml:11]: Correct syntax update for versions.

Comment (chosww) [reply .github/dependabot.yml:11]: When I ran npm install, I was getting deprecated warnings from stylelint, and the warnings were resolved when latest version of stylelint is installed. I am wondering, Is there reason why we are ignoring stylelint update from dependabot?

**Review by chosww — COMMENTED** [review]
Thanks @greatislander, changes look good and works fine. I just left a comment about the syntax for dependabot config you made on other projects.

File: .github/dependabot.yml (hunk 1 above)
Comment (greatislander) [reply .github/dependabot.yml]: See the original PR where I added the dependency restriction: https://github.com/fluid-project/trivet/pull/357. The short answer is that `stylelint-config-fluid` is currently incompatible with newer versions of stylelint.

**Review by chosww — APPROVED** [review]

**Pull request merged**